### PR TITLE
prop: add IsCycleDigraph

### PR DESCRIPTION
**Pull request by james-d-mitchell**
And related documentation and tests.

### CODE REVIEW

File: doc/prop.xml [hunk 1]
```diff
@@ -8,6 +8,37 @@
...
 ##
 
+<#GAPDoc Label="IsCycleDigraph">
+<ManSection>
+  <Prop Name="IsCycleDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    <C>IsCycleDigraph</C> returns <K>true</K> if the digraph <A>digraph</A> is
+    isomorphic to the cycle graph with the same number of vertices as
+    <A>digraph</A>, and <K>false</K> if it is not; ; see <Ref Oper="CycleDigraph"/>.<P/>
```
Comment (mtorpey) [inline doc/prop.xml]: Double semicolon here.

Comment (mtorpey) [inline doc/prop.xml]: Double semicolon here.

**Comment by james-d-mitchell** [comment]
For consistency with `CycleDigraph` I think the 0 vertex, 0 edge graph should not be a cycle digraph. Will amend accordingly.

**Comment by james-d-mitchell** [comment]
I'm not sure why this is failing seems like maybe travis didn't work...

**Comment by james-d-mitchell** [comment]
I got travis to redo the failing test and it failed again, any ideas @mtorpey ??

**Comment by mtorpey** [comment]
Looks like a problem with profiling code - only the coverage checks are failing.  I'm running the tests again.

**Comment by mtorpey** [comment]
All fixed somehow!  Must have been a cosmic ray.

**Pull request merged**